requirements/ci.txt

```diff
@@ -10,7 +10,7 @@ charset-normalizer==2.1.1
     # via requests
 codecov==2.1.12
     # via -r requirements/ci.in
-coverage==6.4.4
+coverage==6.5.0
```

Comment:
No breaking change in the new version: https://github.com/nedbat/coveragepy/blob/master/CHANGES.rst#version-650--2022-09-29